SCClassLibrary/Common/Math/Complex.sc

```diff
@@ -129,6 +137,9 @@ Complex : Number {
 	round { arg aNumber = 1.0;
 		^Complex(real.round(aNumber), imag.round(aNumber))
 	}
+	isInfinite {
```

Comment:
OK, we can do this later when (if) we extend the complex numbers to the Riemann sphere. I just wanted to be explicit.